src/orders/components/OrderCardTitle/OrderCardTitle.tsx

```diff
@@ -130,30 +142,21 @@ const CardTitle: React.FC<CardTitleProps> = ({
     0
   );
 
-  const title = (
-    <>
-      {intl.formatMessage(messageForStatus, {
-        fulfillmentName,
-        quantity: totalQuantity
-      })}
-      {fulfillmentName && (
-        <Typography className={classes.orderNumber} variant="body1">
-          {fulfillmentName}
-        </Typography>
-      )}
-    </>
-  );
-
   return (
     <DefaultCardTitle
       toolbar={toolbar}
       title={
         <div className={classes.title}>
-          {withStatus ? (
-            <Pill label={title} color={selectStatus(status)} />
-          ) : (
-            title
-          )}
+          <div className={classes.indicator}>
+            {withStatus && <CircleIndicator color={selectStatus(status)} />}
+          </div>
```

Comment:
```suggestion
         {withStatus && <div className={classes.indicator}>
            <CircleIndicator color={selectStatus(status)} />
          </div>}
```
Could also include `<HorizontalSpacer spacing={2} />` but I'm not sure about styling